src/Compilers/CSharp/Portable/Parser/Lexer_RawStringLiteral.cs

```diff
@@ -87,7 +87,7 @@ private void ScanRawStringLiteral(ref TokenInfo info, bool inDirective)
             // trusting the contents.
             if (this.HasErrors)
             {
-                var afterStartDelimiter = TextWindow.LexemeStartPosition + startingQuoteCount;
+                var afterStartDelimiter = this.LexemeStartPosition + startingQuoteCount;
```

Comment:
Reference to `textWindow.LexemeStartPosition` were mechanically replaced with `this.LexemeStartPosition`